src/main/java/cn/nukkit/level/Level.java

```diff
@@ -2038,7 +2038,7 @@ public Item useBreakOn(Vector3 vector, int layer, BlockFace face, Item item, Pla
                 }
             }
 
-            double breakTime = target.getBreakTime(item, player);
+            double breakTime = target.calculateBreakTime(item, player);
```

Comment:
Since it has the same signature, is it possible to make `calculateBreakTime` replace `getBreakTime` instead of being a new method?